api/scpca_portal/models/project.py

```diff
@@ -63,6 +62,21 @@ class Meta:
     def __str__(self):
         return f"Project {self.scpca_id}"
 
+    @classmethod
+    def get_from_dict(cls, data: Dict):
+        project = cls(scpca_id=data.pop("scpca_project_id"))
+        # Assign values to remaining properties
+        for key in data.keys():
+            if hasattr(project, key):
+                if key.startswith("includes_") or key.startswith("has_"):
+                    setattr(project, key, utils.boolean_from_string(data.get(key, False)))
+                else:
+                    setattr(project, key, data.get(key))
+
+        project.data_file_paths = project.get_data_file_paths()
+
+        return project
+
```

Comment:
This was not added in this PR, but was moved to the top of the file (similar to the placement of similar `get_from_dict` methods on other models)